CHANGELOG.next.toml

```diff
@@ -79,3 +79,95 @@ message = "Fix cargo audit issue on criterion."
 references = ["smithy-rs#1923"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "ysaito1001"
+
+[[aws-sdk-rust]]
+message = """
+<details>
+<summary>The HTTP connector used when making requests is now configurable.</summary>
+
+```rust
+use std::time::Duration;
+use aws_smithy_client::{Client, hyper_ext};
+use aws_smithy_client::erase::DynConnector;
+use aws_smithy_client::http_connector::ConnectorSettings;
+use aws_types::SdkConfig;
+
+let https_connector = hyper_rustls::HttpsConnectorBuilder::new()
+    .with_webpki_roots()
+    .https_only()
+    .enable_http1()
+    .enable_http2()
+    .build();
+
+let smithy_connector = hyper_ext::Adapter::builder()
+    // Optionally set things like timeouts as well
+    .connector_settings(
+        ConnectorSettings::builder()
+            .connect_timeout(Duration::from_secs(5))
+            .build()
+    )
+    .build(https_connector);
+
+let sdk_config = SdkConfig::builder()
+    .http_connector(smithy_connector)
+    .build();
+
+let client = Client::new(&conf);
+
+// When sent, this operation will go through the custom smithy connector instead of
+// the default HTTP connector.
+let op = client
+    .get_object()
+    .bucket("some-test-bucket")
+    .key("test.txt")
+    .send()
+    .await
+    .unwrap();
+```
+
+</details>
+"""
+references = ["smithy-rs#1225", "smithy-rs#1918"]
+meta = { "breaking" = false, "tada" = true, "bug" = false }
+author = "Velfi"
+
+[[aws-sdk-rust]]
+message = """
+`<service>::Client::from_conf_conn` has been removed since it's now possible to configure the connection from the
+shared and service configs. To update your code, pass connections to the `http_connector` method during config creation.
+
+<details>
+<summary>Example</summary>
+
+before:
+
+```rust
+    let conf = aws_sdk_sts::Config::builder()
```

Comment:
Does `Config` have defaults for the rest of the fields?

Reply:
No, but rather than set them I just added a comment about how it doesn't set them by default but I've omitted them from the example for brevity.